src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -147,7 +147,9 @@ function MoneyRequestView({report, parentReport, parentReportActions, policyCate
         if (!isDistanceRequest) {
             amountDescription += ` • ${translate('iou.cash')}`;
         }
-        if (isCancelled) {
+        if (ReportUtils.isReportApproved(report)) {
```

Comment:
Coming from https://github.com/Expensify/App/issues/35666
As @marcochavezf pointed out in [this](https://github.com/Expensify/App/issues/35666#issuecomment-1932687348) comment, we were getting the 'Approved' status from the wrong report (transaction report) instead of the money request
It should be `ReportUtils.isReportApproved(moneyRequestReport)`